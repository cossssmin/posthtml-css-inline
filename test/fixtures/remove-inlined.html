<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }
  </style>
</head>
<body>
  <div class="text-blue text-sm">text</div>
</body>
</html>
