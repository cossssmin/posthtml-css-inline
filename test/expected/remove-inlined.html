<!DOCTYPE html>
<html>
<head>

</head>
<body>
  <div class="text-blue text-sm" style="font-size: 12px; color: blue">text</div>
</body>
</html>
